.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
- [OPTIONAL] 

- [ ] You can just ignore this file by adding it to the .`gitignore` file because it's not necessary to be in your repository 😄

Reply:
How should I enter that on my .gitignore file is it something like  ".vscode/settings.json" ?

Reply:
Yes 😄